File: .tmp-query1.py
import sqlite3, json
db = sqlite3.connect('db/nco.db')
db.row_factory = sqlite3.Row
rows = db.execute("""
SELECT id, team_id, assigned_to, status, error,
  LENGTH(COALESCE(response,'')) as response_len,
  COALESCE(heartbeat_seq,0) as heartbeat_seq,
  created_at FROM tasks
  WHERE COALESCE(error,'') LIKE '%timeout(idle)%'
    AND LENGTH(COALESCE(response,'')) = 65536
""").fetchall()
print(json.dumps([dict(r) for r in rows], indent=2))
db.close()

File: .tmp-query1.mjs
import Database from 'better-sqlite3';
const db = new Database('db/nco.db',{readonly:true});
const rows = db.prepare(`SELECT id, team_id, assigned_to, status, error,
  LENGTH(COALESCE(response,'')) as response_len,
  COALESCE(heartbeat_seq,0) as heartbeat_seq,
  created_at FROM tasks
  WHERE COALESCE(error,'') LIKE '%timeout(idle)%'
    AND LENGTH(COALESCE(response,'')) = 65536`).all();
console.log(JSON.stringify(rows,null,2));
db.close();
